pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -260,8 +260,15 @@ func (exp *FlowExporter) Run(stopCh <-chan struct{}) {
 func (exp *FlowExporter) sendFlowRecords() (time.Duration, error) {
 	currTime := time.Now()
 	var expireTime1, expireTime2 time.Duration
-	exp.expiredConns, expireTime1 = exp.conntrackConnStore.GetExpiredConns(exp.expiredConns, currTime, maxConnsToExport)
+	// For inter-node traffic with egress/ingress np with action drop, we will receive records from PacketIn and the
+	// conntrack table. For the ingress case, there is no issue as we will receive the records from both nodes and
+	// these two records are both correlationRequired. For the egress case, the record from the conntrack table is
+	// correlationRequired and the record from PacketIn is not correlationRequired. If the record from the conntrack
+	// table arrive at the FA first, then the record will need to do correlation at FA. The ReadyToSend will be true
+	// forever as it keeps waiting to do correlation.
```

Comment:
it is a bit confusing to have references to `correlationRequired` and `ReadyToSend` here as these are internals of the go-ipfix library. Can we express in more generic terms why the FA expects the record from the `denyConnStore` first?

Reply:
But I think the reason why we want to prioritize the record from the `denyConnStore ` is because the `correlationRequired ` behavior is different in two connection_store. For an inter-node service traffic with egress drop np, we will have this issue.

Reply:
I've updated it. Please take a look to see if it is better?